### DISCUSSION TITLE
Lock STIG Version

### DISCUSSION THREAD

**Discussion by TheTroubleshooter** [NUWCDIVNPT/stig-manager#745] [Feedback Requested]
Is there a way to lock the version of the checklist to an older STIG version? When using the change version explained in the documentation the version change does not persist after exiting the tab. Most of my work is project based so having a historical repository is important for old project collections and I do not want those old project collections to update to the new STIG's versions when they are imported quarterly.

**Comment by cd-rite** [comment]
Hi @TheTroubleshooter Thanks for the suggestion! We have considered adding a feature like this, but had not heard much demand for it yet. Original Issue here: #194 

Thanks for starting the discussion.... We will consider your interest when prioritizing feature implementations.

**Reply by TheTroubleshooter** [reply]
This would be a good feature for auditors that mainly perform project based work. I appreciate the response and thank you for linking me to the prior request. I will do my best to search for different words in the future.

**Comment by cd-rite** [comment]
Hi @TheTroubleshooter We added support for letting Owners and Managers "pin" their Collection's STIGs to specific Revisions with Release 1.3.8:
https://github.com/NUWCDIVNPT/stig-manager/releases/tag/1.3.8
